core_tic_tac_toe/display.py

```diff
@@ -48,19 +46,30 @@ def clear_console(self):
     def print_invalid_game_selection_message(self):
         self.io.print_message("Invalid option, please try again!")
 
-    def __horizontal_bar(self):
-        return "-----------------\n"
+    def __generate_horizontal_bars(self, move):
+        return ("-----------" * len(move)) + "\n"
 
-    def __vertical_bars(self):
-        return "     |     |    \n"
+    def __generate_vertical_bars(self, move):
+        vertical_bars = ""
+
+        for idx in range(0, len(move)):
+            if idx != len(move) - 1:
+                vertical_bars += "          |"
+            else:
+                vertical_bars += "          \n"
+
+        return vertical_bars
 
     def __generate_rows(self, move):
         row = ""
 
         for idx, position in enumerate(move):
             if idx != len(move) - 1:
-                row += "  {}  |".format(position)
+                if type(position) == int and position // 10 > 0:
```

Comment:
Why `10`?

Reply:
The format for double digit numbers is different than the format for single digit numbers. I'll go ahead and extract that piece of logic into its own method so I can give it a more descriptive name.